EIPS/eip-2786.md

```diff
@@ -12,38 +12,60 @@ requires: 2700
 ---
 
 ## Simple Summary
+
 When an Ethereum Provider becomes connected or disconnected, it will emit a `connect`/`disconnect` event.
 
 ## Abstract
+
 The Provider is said to be “connected” when it can service RPC requests to at least one chain.
 The Provider is said to be “disconnected” when it cannot service RPC requests to any chain at all.
 When the Provider switches from a "connected" state to a "disconnected" state, it will emit a `connect` event.
 When the Provider switches from a "disconnected" state to a "connected" state, it will emit a `disconnect` event.
 
 ## Motivation
+
 When an application is hooked up to an Ethereum provider, there is value in having the application be alerted of connect/disconnect events that may occur so the application can appropriately inform the user of the situation.
 It is left up to the application to decide whether to listen in on these events, and how to handle them.
 
 ## Specification
-<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
 ### Definitions
+
 #### Connected
+
 The Provider is considered `connected` when it is able to service RPC requests to at least one chain.
+
 #### Disconnected
+
 The Provider is considered `disconnected` when it is unable to service RPC requests to any chain.
+
 ### Events
+
 #### `connect`
+
 The Provider **MUST** emit a `connect` event to all attached [EIP-2700](./eip-2700) listeners if it transitions from a `disconnected` state to a `connected` state.
-All attached listeners **MUST** be called with the parameters `('connected', { chainId })`.
+All attached listeners **MUST** be called with the parameter `{ chainId }`.
```

Comment:
I don't think this change is correct.  The attached listener is a function that takes two parameters, first being a string second being dependent on the first.  If you were to call an attached listener with just `{ chainId }` that would be incorrect behavior.

Reply:
Bleh, you were right, I was wrong.